pkg/controllers/user/pipeline/engine/jenkins/engine.go

```diff
@@ -70,7 +70,12 @@ func (j *Engine) PreCheck() error {
 		return err
 	}
 	user := JenkinsDefaultUser
-	token := JenkinsDefaultToken
+	secret, err := j.Core.Secrets(utils.PipelineNamespace).Get("jenkins", metav1.GetOptions{})
```

Comment:
Maybe we should figure out why cache for Secret does not work.

Reply:
I don't reproduce secret lister issue so I will update to use lister here.